frame/balances/call_set_balance.go

```diff
@@ -31,13 +31,26 @@ func newCallSetBalance(moduleId sc.U8, functionId sc.U8, storedMap types.StoredM
 	return call
 }
 
-func (c callSetBalance) DecodeArgs(buffer *bytes.Buffer) types.Call {
+func (c callSetBalance) DecodeArgs(buffer *bytes.Buffer) (types.Call, error) {
+	addr, err := types.DecodeMultiAddress(buffer)
+	if err != nil {
+		return nil, err
+	}
+	compact1, err := sc.DecodeCompact(buffer)
+	if err != nil {
+		return nil, err
+	}
+	compact2, err := sc.DecodeCompact(buffer)
```

Comment:
Can we name all decode arguments as what their purpose is: `targetAddress`, `newFree` and `newReserved`